spec/helpers/sufia_helper_spec.rb

```diff
@@ -264,4 +264,10 @@ def create_models(model, user1, user2)
       )).to eq("<a href=\"http://creativecommons.org/publicdomain/zero/1.0/\">CC0 1.0 Universal</a>, <a href=\"http://creativecommons.org/publicdomain/mark/1.0/\">Public Domain Mark 1.0</a>, and <a href=\"http://www.europeana.eu/portal/rights/rr-r.html\">All rights reserved</a>")
     end
   end
+
+  describe "#human_readable_date" do
+    it "ensures that the display of the date is human-readable" do
+      expect(helper.human_readable_date(value: [DateTime.new(2016, 8, 15)])).to eq("08/15/2016")
```

Comment:
The build is failing because of rubocop here. Instead of calling `DateTime.new()`, why not pass in the string literal? Like:

```ruby
expect(helper.human_readable_date(value: ["2016-08-15T00:00:00Z"])).to eq("08/15/2016")
```
?

Then I'll :shipit:. Thanks!